etna/people/models.py

```diff
@@ -57,24 +64,41 @@ class PersonPage(BasePage):
     to link pages to a person, or as a reference to the person.
     """
 
-    role = models.CharField(blank=True, null=True, max_length=100)
-    summary = RichTextField(
-        blank=True, null=True, features=settings.RESTRICTED_RICH_TEXT_FEATURES
-    )
     image = models.ForeignKey(
         get_image_model_string(),
         blank=True,
         null=True,
         on_delete=models.SET_NULL,
         related_name="+",
     )
+    role = models.CharField(blank=True, null=True, max_length=100)
+    summary = RichTextField(
+        blank=True, null=True, features=settings.RESTRICTED_RICH_TEXT_FEATURES
+    )
+
+    research_summary = StreamField(ResearchSummaryStreamBlock, blank=True, null=True)
```

Comment:
Is this the structure the content team wanted?

I was under the impression it was something more formulaic like:

- `research_activity` - optional single richtext
- `select_publications` - optional list of richtext fields
- `specialism` - optional list of text fields

Reply:
That's cool then. Thank you for checking 👍🏼